microsoft-dynamics-crm/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -19,7 +19,12 @@ To use this connector, you must be familiar with:
 
 Before creating an app, you must have login credentials for Microsoft Dynamics CRM SOAP Organization Service to test your connection to your target resource.
 
-Note that you must format a date in milliseconds with a maximum 3 of digits to the right of the decimal.
+Any created or expired date in the web service header created by the Microsoft CRM instance must respect the constraints specified in the BasicSecurity Profile validation for Web Services, `http://www.ws-i.org/Profiles/BasicSecurityProfile-1.0.html`, as summarized below:
+
+* A created or expired date must not contain a seconds value with more than three digits to the right of the decimal (milliseconds).
+* A created or expired date that contains a seconds value must specify a seconds value less than 60.
+* A created or expired date must not include the ValueType attribute.
+* A created or expired date must contain time values in UTC format, as specified by the XML Schema type (dateTime).
```

Comment:
It seems like maybe the last bullet should be the first bullet because it's talking about the required format for the values.

Reply:
This is the order in the spec.